ada_feeding/ada_feeding/behaviors/acquisition/compute_action_constraints.py

```diff
@@ -102,6 +133,15 @@ def update(self) -> py_trees.common.Status:
         index = np.random.choice(np.arange(prob.size), p=prob)
         action = response.actions[index]
 
+        ### Calculate Approach Frame
+        # TODO: Calculate Approach Frame
+        identity = TransformStamped()
+        identity.transform = ros2_numpy.msgify(Transform, np.eye(4))
+        identity.header.stamp = self.node.get_clock().now().to_msg()
+        identity.header.frame_id = "food"
+        identity.child_frame_id = "approach"
+        set_static_tf(identity, self.blackboard, self.node)
```

Comment:
Just checking that the approach frame is only the same as the food frame for action 0, and this will be changed for the general implementation, right?

Reply:
Yes, that's the TODO. This PR is only guaranteed valid for action 0